src/local/LocalTransport.ts

```diff
@@ -8,9 +8,16 @@ export interface Listener<T> {
   (message: T): void;
 }
 
-export default class LocalTransport {
+export interface ITransport {
```

Comment:
It's not idiomatic to prefix interfaces with `I` in TypeScript. It is just a `Transport`. `LocalTransport` is a realization of `Transport`.